code/API_definitions/population-density-data.yaml

```diff
@@ -94,37 +94,31 @@ info:
-    Which specific authorization flows are to be used is determined during
-    onboarding process, happening between the API Client and the MNO 
-    exposing the API, taking into account the declared purpose for accessing the API,
-    while also being subject to the prevailing legal framework dictated by local legislation.
-    It is important to remark that in cases where personal user data is processed by the API,
-    and users can exercise their rights through mechanisms such as opt-in and/or opt-out,
-    the use of 3-legged access tokens becomes mandatory. This measure ensures that the API remains
-    in strict compliance with user privacy preferences and regulatory obligations, upholding the 
-    principles of transparency and user-centric data control.
-
-    Population Density Data API ensures the usage of anonymized information and do not treat
-    personal data neither as input nor output. 
-    Therefore, the access to Population Density Data API is defined as 
-    Client Credentials - 2-legged as described in [CAMARA-API-access-and-user-consent.md - Client Credentials](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#client-credentials)
+    The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
+
+    Which specific authorization flows are to be used will be determined during onboarding process, happening between the API Client and the Telco Operator exposing the API, taking into account the declared purpose for accessing the API, while also being subject to the prevailing legal framework dictated by local legislation.
+
+    It is important to remark that in cases where personal user data is processed by the API, and users can exercise their rights through mechanisms such as opt-in and/or opt-out, the use of 3-legged access tokens becomes mandatory. This measure ensures that the API remains in strict compliance with user privacy preferences and regulatory obligations, upholding the principles of transparency and user-centric data control.
+
+    Population Density Data API ensures the usage of anonymized information and do not treat personal data neither as input nor output. 
+    Therefore, the access to Population Density Data API is defined as Client Credentials - 2-legged. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the latest detailed specification of this authentication/authorization flow.
+
 
     # Further info and support
 
-  termsOfService: http://swagger.io/terms/
-  contact:
-    email: project-email@sample.com
+    (FAQs will be added in a later version of the documentation)

```

Comment:
why add "TBD" entry to the released version of the spec?

Reply:
This line was copied over from the very first API documentations. It was never mandatory, even not in the outdated documentation template (which will now get [deprecated](https://github.com/camaraproject/Commonalities/pull/261)). So you can delete this line.

Reply:
I have removed it PR #42 . And now it is added back.